### PR TITLE
Switch messages to sever and minor CSS adjustment

### PR DESCRIPTION
**Pull request by alshakerM**
Switched messages to sever

### CODE REVIEW

**Comment by alshakerM** [comment]
Added margin to user header

File: components/pages/Messages/Messages.js [hunk 1]
```diff
@@ -80,6 +80,7 @@ export function Messages({ fromUserId }) {
   const messagesData = useSelect((select) =>
     select('ig-messages').getMessages()
   );
+  console.log(messagesData)
```
Comment (alshakero) [inline components/pages/Messages/Messages.js:83]:
```suggestion
 
```

**Review by alshakero — APPROVED** [review]
This change looks good. But it's still loading all messages of all users. So far, we have only 20 users, but soon we'll have many. I think we should implement fetching messages from one user. 

It's up to you, you can merge this PR and make another one, or continue here.

File: components/pages/Messages/Messages.js (hunk 1 above)
Comment (alshakerM) [reply components/pages/Messages/Messages.js]: Done

**Pull request merged**